### ISSUE TITLE
Multiple Images and multiple icons

### ISSUE THREAD

**Issue by mrkevanlin** [preciousforever/sketch-data-populator#60]
Sketch crashes whenever I try to populate my field group with more than one image and with more than one svg. I have a product image, an image of ratings, and two svg's for two badges. Is there support or help for this? Thanks!

**Comment by nitrada** [comment]
Can you send JSON and sketch file to feedback@datapopulator.com?

**Comment by mrkevanlin** [comment]
Due to the sensitive nature of my work I can't actually upload these assets.

**Comment by nitrada** [comment]
@mrkevanlin understandable. can you try populating a) without SVG (that feature is shaky) and b) just one SVG per group?

**Comment by mrkevanlin** [comment]
Just to make sure that I could get up and running with this plugin I was only working with two sets of data in my JSON file. In my sketch file, I indicated to duplicate 2 rows of data. 

Tried to populate with SVG on both sets (duplicate twice) - crash. 
Tried to populate without SVG on both sets - worked! 
Did some testing, and made my product image and one of the badges both jpgs - crashed. When I switched one of them to png - worked.
